test/built-ins/RegExp/prototype/Symbol.matchAll/this-val-non-obj.js

```diff
@@ -1,36 +0,0 @@
-// Copyright (C) 2018 Jordan Harband. All rights reserved.
```

Comment:
This test is now: [this-not-object-throws.js](https://github.com/peterwmwong/test262/blob/matchall-pww/test/built-ins/RegExp/prototype/Symbol.matchAll/this-not-object-throws.js)
  - Updated according to: https://github.com/tc39/proposal-string-matchall/pull/33